docs/ref/settings.txt

```diff
@@ -3001,9 +2995,7 @@ useful way to mitigate the risk of a client side script accessing the
 protected cookie data.
 
 Turning it on makes it less trivial for an attacker to escalate a cross-site
-scripting vulnerability into full hijacking of a user's session. There's not
-much excuse for leaving this off, either: if your code depends on reading
-session cookies from JavaScript, you're probably doing it wrong.
+scripting vulnerability into full hijacking of a user's session.
 
 .. _HTTPOnly: https://www.owasp.org/index.php/HTTPOnly
```

Comment:
This link looks very out of date, containing only [minor changes since 2014](https://www.owasp.org/index.php?title=HttpOnly&type=revision&diff=232658&oldid=185263). Perhaps https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#Secure_and_HttpOnly_cookies would be more appropriate if we keep the setting?

It feels like this is getting out of scope of this PR though, so perhaps creating a ticket would be best.

Reply:
There are a bunch of old links like this, some needing minor updates e.g. to be HTTPS over HTTP, others like this probably better to re-point. I think it would be a separate ticket?

Reply:
I could try and do that. If it's better to have a seperate ticket would you assign it to me?

Reply:
I've created https://code.djangoproject.com/ticket/29784.

Reply:
@IgnatiusFlo Ticket is accepted. If you want to grab it that would be fab! (c.f. #10430)

Reply:
Hey, Sure. Sorry. It looked like someone had taken it. I won't be in front of my computer till tomorrow (no allowed python on my work PC), so bear with me....

Reply:
Or can I just edit it in my fork? Do I need anything else?